packages/volto/news/7023.documentation

```diff
@@ -0,0 +1,3 @@
+Corrected file references in Redux-related docs, improved word wrapping to
+column 80 in source code of Redux-related docs and added link to Redux
+development addon for Firefox. @silviubogan
```

Comment:
Please follow line wrapping guidance from [Plone documentation styleguide](https://6.docs.plone.org/contributing/documentation/authors.html#plone-documentation-styleguide). Also please read the explanation for this specific guidance.

> - Use one sentence per line.